src/System Application/Test/Azure File Services API/src/AzureTestHelper.Codeunit.al

```diff
@@ -0,0 +1,82 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.Azure.Storage.Files;
+
+using System.Azure.Storage.Files;
+
+/// <summary>
+/// Provides common authorization functionality for using Azure storage accounts in tests for Azure Storage Services.
```

Comment:
does it provide common auth functionaliy? As far as i see most of the code clears the fileshare (i.e. a reset or init for tests).

Reply:
Right, I updated the summary.